.project

```diff
@@ -5,13 +5,19 @@
 	<projects>
 	</projects>
 	<buildSpec>
+		<buildCommand>
```

Comment:
VSCode did this

Reply:
delete the file, we don't need it and shouldn't be there